File: DSA/GEHNA_KHANNA_590013140/day1question3.c
#include<stdio.h>
int main()
{
    int n;
    int sum=0;
    printf("enter array size\n");
    scanf("%d",&n);
    int arr[n];
    printf("enter array elements\n");
    for(int i=0;i<n;i++)
    {
        scanf("%d",&arr[i]);
        sum+=arr[i];
    }
    printf("\n");
    printf("sum of array is %d\n",sum);
}

File: DSA/VaniJain_590012020/Day1Question3.java
import java.util.*;
public class Day1Question3{
    public static void main(String[] args){
    System.out.println("Enter the no. of elements in the aaray:")
    Scanner scr=new Scanner(System.in);
    int n=scr.nextInt();
    int [] arr=new int[n];
    int i;
    System.out.println("Enter the elements into the array:");
    for(i=0;i<n;i++)
    {
        arr[i]=scr.nextInt();
    }
    int sum=0;
    for(i=0;i<n;i++)
    {
        sum=sum+arr[i];
    }
    System.out.println("The sum of the array is "+sum);
}
}
